LiftLog.Ui/Models/SessionHistoryDao/SessionHistoryDaoV2.proto

```diff
@@ -34,5 +34,6 @@ message PotentialSetDaoV2 {
 
 message RecordedSetDaoV2 {
     int32 reps_completed = 1;
-    LiftLog.Ui.Models.TimeOnlyDao completion_time = 2;
+    LiftLog.Ui.Models.DateOnlyDao completion_date = 2;
+    LiftLog.Ui.Models.TimeOnlyDao completion_time = 3;
```

Comment:
```suggestion
    LiftLog.Ui.Models.TimeOnlyDao completion_time = 2;
    optional LiftLog.Ui.Models.DateOnlyDao completion_date = 3;
```

Protobuf is an interesting format in that to maintain backwards compatibility, existing fields should have the same index (i.e. `completion_time` should remain at index `2`).  I have also marked completionDate as optional, as we may not have it on older saves